test/devices_tests/sensor_test.py

```diff
@@ -79,6 +79,11 @@ async def async_after_update_callback(device):
         sensor.sensor_value.payload = payload
 
         telegram = Telegram(group_address=GroupAddress("1/2/3"), payload=payload)
+        response_telegram = Telegram(
```

Comment:
Why did you change this test?

Reply:
So it can use process() instead of directly process_group_response() like it is done in production by the TelegramQueue. (line 101)